web/thesauruses/javascript/ECMAScript 2009/functions.json

```diff
@@ -0,0 +1,113 @@
+{
+  "meta": {
+    "language": "javascript",
+    "language_name": "JavaScript",
+    "structure": "functions",
+    "language_version": "ECMAScript 2009"
+  },
+  "concepts": {
+    "void_function_no_parameters": {
+      "name": "Function that does not return a value and takes no parameters",
+      "code": "function method() {}"
```

Comment:
Typically I see JS functions on multiple lines (like
```
function method() {
   statement;
}
```

While I know JS doesn't care about whitespace, it's typically the "style" of the language. Would you mind adding `\n` characters to separate out lines, or just put separate lines as part of the `code: []` array?